server/routes/eventAPI.js

```diff
@@ -20,7 +20,7 @@ router.post("/api/event", auth, (req, res) => {
 
 // GET all meetings for user
 router.get("/api/event", auth, (req, res) => {
-    db.EventType.find({ user_id: req.userId })
+    db.EventType.find({ user_id: req.user_id })
```

Comment:
I think what Dan meant is to use userId instead of user_id

Reply:
https://github.com/hatchways/team-yugioh/pull/58#discussion_r560954725
It originally was userId and he left this comment. He does say snakecase to camelcase but I assumed he meant to change req.userId as [type]_id is what we've been using in all of our models and what was specified in the issues about creating them

Reply:
I think he was referring to the former part. The database key should be `userId` instead of `user_id`? Let's just clarify that with him in the standup
` db.EventType.create({ ...req.body, user_id: req.userId })`